framework/platform/wayland/tcuWayland.hpp

```diff
@@ -0,0 +1,129 @@
+#ifndef _TCUWAYLAND_HPP
+#define _TCUWAYLAND_HPP
+/*-------------------------------------------------------------------------
+ * drawElements Quality Program Tester Core
+ * ----------------------------------------
+ *
+ * Copyright 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *//*!
+ * \file
+ * \brief wayland utilities.
+ *//*--------------------------------------------------------------------*/
+
+#include "tcuDefs.hpp"
+#include "gluRenderConfig.hpp"
+#include "gluPlatform.hpp"
+#include "deMutex.hpp"
+
+#include <wayland-client.h>
+#include <wayland-egl.h>
+
+#ifdef __GNUC__
```

Comment:
Is UNUSED used somewhere? If you have unused args, just leave name out, or add DE_UNREF(unusedArg) in func body.